components/backends/trtllm/multinode/start_trtllm_worker.sh

```diff
@@ -39,7 +39,7 @@ if [[ -n ${DISAGGREGATION_STRATEGY} ]]; then
 fi
 
 trtllm-llmapi-launch \
-  python3 /mnt/components/worker.py \
+  python3 -m dynamo.trtllm.worker \
```

Comment:
```suggestion
  python3 -m dynamo.trtllm \
```

Reply:
Fixed via: [Address the review comments](https://github.com/ai-dynamo/dynamo/pull/2003/commits/42fa8983bf1580b1bc12b587a0113d4fe23de6bf)